std/StandardToken.sol

```diff
@@ -3,20 +3,20 @@ pragma solidity ^0.4.0;
 import "./Token.sol";
 
 contract StandardToken is Token {
-	uint256 public totalSupply;
```

Comment:
@chriseth I've chosen not to change the standard token contract.  My opinions change because I'm still developing my mental model about Solidity.